opentreemap/importer/models/base.py

```diff
@@ -30,6 +31,15 @@ class Meta:
     FAILED_FILE_VERIFICATION = 6
     CANCELED = 8
     VERIFICATION_ERROR = 10
+    UNKNOWN_ERROR = 12
+
+    @classproperty
+    def INACTIVE_CODES(self):
```

Comment:
This method produces a list of codes that are used by predicate-style named methods (eg. `is_finished`), of which this class has several. For that reason, I think giving it a non-predicate-style name is a good idea, which is why I chose the `_CODES` suffix.

I do agree with your point. How about `INACTIVE_STATUS_CODES`?